lib/stat_tracker.rb

```diff
@@ -113,4 +113,47 @@ def worst_coach(season)
 
     coach_results.min_by { |k, v| v }[0]
   end
+
+  def calculate_team_accuracies(season)
+    team_accuracies = {}
+
```

Comment:
please add a comment here what game_tams_in_season 'shape' is, what is the expected data structure to look like?

Reply:
Done!